code/controlconfig/controlsconfig.cpp

```diff
@@ -171,14 +173,18 @@ static int Background_bitmap;
 static int Conflicts_tabs[NUM_TABS];
 static UI_BUTTON List_buttons[LIST_BUTTONS_MAX];  // buttons for each line of text in list
 static UI_WINDOW Ui_window;
-static unsigned int Defaults_cycle_pos; // the controls preset that was last selected
+static unsigned int Defaults_cycle_pos = 0; // the controls preset that was last selected
 
 int Control_config_overlay_id;
 
-static struct {
+struct conflict {
 	int key;  // index of other control in conflict with this one
 	int joy;  // index of other control in conflict with this one
```

Comment:
You can initialize the members directly as well:

```suggestion
	int key = -1;  // index of other control in conflict with this one
	int joy = -1;  // index of other control in conflict with this one
```
Then you do not need to explicitly declare the default constructor.

Reply:
I think one of the CI's complained when I did that, most likely clang tidy.